src/api/users/service/users.service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@nestjs/common';
+import { UserRepository } from '../repository/users.repository';
+
+@Injectable()
+export class UserService {
+  constructor(private readonly userRepo: UserRepository) {}
+  async getAllUsers() {
+    return await this.userRepo.getAllUsers();
```

Comment:
return this.userRepo.getAllUsers();

await key won't have effect if the method is asny.. by default it will return async
But, if we only need when ... const user = await this.repo.someting()